src/style.css

```diff
@@ -0,0 +1,34 @@
+body {
+  margin: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
+  background-size: cover;
+  display: flex;
+  width: 12px;
+  height: 12px;
+  margin: 2px 6px;
+  margin-left: 2px;
```

Comment:
According to the mockup, you should only add margin-right to all stars except the last stars in a row
<img width="415" alt="Screenshot 2023-09-14 at 12 31 12" src="https://github.com/mate-academy/layout_stars/assets/80700247/5d1ad803-a7c5-48f3-b81c-d3a55b67dad7">